tests/compute/test_sampling.py

```diff
@@ -1026,12 +1059,15 @@ def test_global_uniform_negative_sampling(dtype):
 
 if __name__ == '__main__':
     from itertools import product
+    test_sample_neighbors_noprob()
+    test_sample_neighbors_prob()
+    test_sample_neighbors_mask()
     for args in product(['coo', 'csr', 'csc'], ['in', 'out'], [False, True]):
         test_sample_neighbors_etype_homogeneous(*args)
-    test_non_uniform_random_walk()
+    test_non_uniform_random_walk(False)
     test_uniform_random_walk(False)
     test_pack_traces()
-    test_pinsage_sampling()
+    test_pinsage_sampling(False)
```

Comment:
Do test functions defined under `if __name__ == '__main__':` still come into effect now? @Rhett-Ying

Reply:
It doesn't for pytest, but it does if I run it in standalone `python` for debugging.

Reply:
I usually run specific testcase via `-m pytest tests/compute/test_sampling.py::test_sample_xx`. why do we need to run it as standalone python script?

Reply:
Ah yeah you are right.  `__main__` is no longer needed then.  We should document this though.